tools/sgterraform/tools.go

```diff
@@ -0,0 +1,52 @@
+package sgterraform
+
+import (
+	"context"
+	"fmt"
+	"os/exec"
+	"path/filepath"
+	"runtime"
+
+	"go.einride.tech/sage/sg"
+	"go.einride.tech/sage/sgtool"
+)
+
+const (
+	version    = "1.1.4"
+	binaryName = "terraform"
+)
+
+func Command(ctx context.Context, args ...string) *exec.Cmd {
+	return sg.Command(ctx, filepath.Join(binDir(), binaryName), args...)
```

Comment:
Bindir can be replaced with `sg.FromBinDir(binaryName)` and with that you can also remove the binDir function

Reply:
Hurray for collaboration. Fixed. Too bad it can't be used in a constant